src/app/item-popup/ItemStat.tsx

```diff
@@ -104,6 +126,26 @@ export default function ItemStat({ stat, item }: { stat: DimStat; item: DimItem
           </div>
         </div>
       )}
+
+      {!!totalDetails && (
+        <div className={styles.totalStatDetailed}>
+          <span>{totalDetails.baseTotalValue}</span>
+          {!!totalDetails.totalModsValue && (
+            <>
+              {' + '}
```

Comment:
Could you include the `+` in the colored span?

Reply:
Sure can. I thought it looked a little weird personally but happy to do it.